update(readme): various enhancements

salmad3 opened the pull request:
Iterates on the existing content of the README, focussing on _Preamble_, _Declaration of Genesis_, _Vision and Missions_, and _Terms_.

- Aims to iterate on writing style for clarity and impact.
- Aims to incorporate key AtomOne propositions through keyword usage (e.g., protect, necessary).
- Initiated content modularization: created 'MILESTONES.md', 'FAQs.md', and 'TO-DO.md' from the README.
- Implements formatting with a heading hierarchy.
  > Perhaps ToC is not necessary as there is a built-in feature (right-hand corner) now on GitHub that displays the ToC. May be useful for those using an editor.
- Relocates the 'Terms" section to the top of the document as a dropdown menu.
- 'Hub' as a proper noun?

If this approach seems okay, we can continue on with the other sections.

Looking forward to your feedback and suggestions.

salmad3 committed:
update(readme): enh to preamble, dec. of genesis

salmad3 committed:
update(readme): enh vision and mission, mv terms to dropdown, init readme header + edits

salmad3 committed:
feat: init temp to-dos doc

salmad3 committed:
feat: init faq doc

salmad3 committed:
feat: init milestones doc

salmad3 committed:
edits

moul commented on TO-DO.md:
```diff
@@ -0,0 +1,36 @@
+# General TODOs & Informal Notes
```
The README should include TODO items to ensure that a single version of the document reflects both the current state of the project and the pending tasks.

salmad3 replied:
Meaning these should be kept in the README and we should remove the proposed TO-DO file?

moul commented on README.md:
```diff
@@ -1,122 +1,214 @@
+
+<!-- TOC -->
+
+- [Preamble](#preamble)
```
TOC is good, is it generated by a tool?
Could be nice to have a makefile rule that automatically add TOC on various docs.

salmad3 replied:
Used a built-in feature with vs code. There are tools available for this too.

salmad3 committed:
edits

moul commented on README.md:
```diff
@@ -1,122 +1,214 @@
-_This document is a work in progress, please make PRs_ _What matters at this
```
This disclaimer should be kept.

salmad3 replied:
Ah, merged it with the other disclaimer to create one, [here](https://github.com/atomone-hub/genesis/pull/92/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R14-R17).

Should it be kept separate?

moul replied:
The README should include a disclaimer and serve as a self-contained document. It should also contain a todo list. This way, regardless of the version, you can read the README and find the disclaimer (if applicable), the current content, and the todo list.

salmad3 replied:
@moul addressed.

moul commented on README.md:
```diff
@@ -139,7 +231,7 @@ pegging and using these $ATOMs to make voting decisions on Cosmos as a voting
 use $ATOMs; by offering what is misleadingly referred to as "liquid staking".
 
-## Role as $ATOM "Liquid Staking" Provider
+### AtomOne's Strategy for $ATOM Bonding and Liquid Staking
```
Was better before; not it’s inconsistent (mixing with and without AtomOne in the titles)

moul commented on README.md:
```diff
@@ -1,122 +1,214 @@
 party (which came about first) is well organized in comparison, usually behind
 closed doors. Meanwhile "liquid staking" providers are providing a service that
-does more than liquid staking, but have their own governance and powers and
+does more than liquid staking, but has their own governance and powers and
```
```suggestion
does more than liquid staking, but have their own governance and powers and
```

Subject is: “liquid staking providers”

salmad3 replied:
Thanks - haven't yet iterated on this section

salmad3 committed:
address PR feedback

salmad3 commented:
@moul gentle ping

salmad3 committed:
update(readme): keep to-dos in readme, sep. warnings

moul commented on MILESTONES.md:
```diff
@@ -0,0 +1,50 @@
+# Milestones
```
Could you please move the milestones back in the main readme?

One reason is that it provides a high-level overview and makes the readme more self-explanatory.

Additionally, we can have separate .md files for sub-projects that can have their own milestones.

moul commented on README.md:
```diff
@@ -1,122 +1,215 @@
+> **This document is a work in progress. We encourage you to make pull requests. Currently, our priority is to open issues and initiate discussions about every facet that should be included in the founding documents.**
+
+<p align="left">
+  AtomOne is a strategic fork of the Cosmos Hub. It redefines the Cosmos 
```
"it redefines the cosmos ecosystem as a minimal IBC/ICS Hub." -> not correct/accurate.

moul commented on README.md:
```diff
@@ -1,122 +1,215 @@
+
+<p align="left">
+  AtomOne is a strategic fork of the Cosmos Hub. It redefines the Cosmos 
+  ecosystem as a minimal IBC/ICS Hub. AtomOne is committed to secure, decentralized 
```
"AtomeOne is committed" -> The project isn't committed by him alone. Perhaps the community or authors are, but not the project itself.

moul commented on README.md:
```diff
@@ -1,122 +1,215 @@
+<p align="left">
+  AtomOne is a strategic fork of the Cosmos Hub. It redefines the Cosmos 
+  ecosystem as a minimal IBC/ICS Hub. AtomOne is committed to secure, decentralized 
+  decision-making, and focuses on driving innovative advancements within the Cosmos 
```
The focus is not on driving, but on enabling. We prioritize creating a secure minimal hub that unlocks more opportunities for experimentation with fewer risks.

moul commented on README.md:
```diff
@@ -1,122 +1,215 @@
+AtomOne emerges as a necessary alternative, accounting for contingencies and 
+prepared for challenges, while providing a haven for various political ideas.
+
+<details>
+<summary>Terms</summary>
```
```suggestion
### Terms
```

moul commented on README.md:
```diff
@@ -1,122 +1,215 @@
+IBC/ICS Hub secured by a staking token, aiming for two-thirds of the stake to be 
+bonded.**
+
+While AtomOne is dedicated to guiding Gaia toward prudent decisions, it cannot 
+unilaterally dictate the choices of the Cosmos Hub. The Cosmos ecosystem requires 
+clear, minimal IBC/ICS hubs, a role that Gaia has not fully embraced, as outlined 
+in the [Declaration of Genesis](declaration-of-genesis). Therefore, AtomOne's 
+mission extends beyond merely advising Gaia; it also involves operating as an 
+essential alternative minimal IBC/ICS hub, complementing Gaia's function within 
+the ecosystem.
```
If I'm not mistaken, this appears to be a new version of this.

```
While AtomOne aims to steer Gaia toward safe decisions, it cannot by itself
determine the decisions made by the Cosmos Hub. Yet, Cosmos *needs* minimal
IBC/ICS hubs without any confusion about what it is, and as mentioned in the
Declaration of Genesis Gaia does not embody this (yet). Ergo, AtomOne must not
only guide Gaia, but it must also run the desired alternative minimal IBC/ICS
hub as an alternative in addition to Gaia.
```

I believe it is unwise to submit pull requests that rewrite everything, potentially using ChatGPT, and eliminate the concept of a changelog diff. Git did not recognize this, but I did, using my human eyes.

Since I must rely on my human eyes and you have altered everything, it is now my responsibility to verify that your ChatGPT generation has not altered the meaning.

Please refer to https://github.com/atomone-hub/genesis/issues/39.

I think the only justification for accepting such changes is if you, as the author, assume full responsibility for limiting the size of the diff and ensuring that the content remains accurate and uses efficient language.

salmad3 replied:
The changes were done line by line on my own. I did not rely on generative AI. Certain terminology is intentional, according to my understanding of what the statement is conveying.

The genesis document is an official document - subject to interpretation and should be written as such.

jaekwon replied:
I'll come back to look at this soon.

moul reviewed (changes requested):
This pull request is too large for minimal improvements. Additionally, the changelog is lost because most of the text appears to have been rewritten, possibly by an AI.

I will close it, and you can reopen smaller pull requests if certain parts of this one should be merged.

salmad3 commented:
> This pull request is too large for minimal improvements. Additionally, the changelog is lost because most of the text appears to have been rewritten, possibly by an AI.
> 
> I will close it, and you can reopen smaller pull requests if certain parts of this one should be merged.

Context provided [here](https://github.com/atomone-hub/genesis/pull/92/files#r1476183279). There are also general grammatical errors beyond {re-}writing the content to add a layer of formality.

salmad3 commented:
Thanks for the input, @moul.
 
- The PR can be split, not sure why we went through three reviews to suggest this. 
- MILESTONES.md was created as a result of this suggestion.
- Suggestions for the Introduction of the README are received.

Above all, the claim on the proposed writing style improvements is misleading, and, like the first point, unsure why we went through three reviews to speculate this. 

> ...eliminate the concept of a changelog diff. Git did not recognize this, but I did, using my human eyes.

Contribution split is useful - I believe the diff is not directly available in that section of the README due to content being moved around (such as the terms.)

> Since I must rely on my human eyes and you have altered everything, it is now my responsibility to verify that your ChatGPT generation has not altered the meaning.

Points of contention should be clearly pointed out as opposed to speculating. I believe there are also verification methods available.

I've provided an example below. If necessary, I can be abundantly clear and provide detailed context for each phrase and statement that was proposed in this PR, and detail the overall relation to AtomOne x Genesis.

_Keep in mind that I'm providing context to back the efforts of the contributions, having made them in good faith...
The actual contributions, as mentioned in the PR description, are iterative and encourage input; the writing style is merely a suggestion._

## Example using "Declaration of Genesis"

General:

- Nature of the document requires a consistent layer of formality
- Mix of past/present tense
- Lack of flow
- Acknowledging that this is an opinionated document, it should still aim to state things positively

So iterating entirely on the existing content felt appropriate. 

### P1:

```
There comes a time when there is enough disagreement among community members and stakers about key concerns regarding the business of their chain, such as its vision, mission, tokenomics, architecture, implementation, or philosophy; that it makes the most sense to support an alternative fork running alongside the original so as to be prepared against all contingencies.
```

- Unclear and complex sentence structure
- Improper use of a semicolon that should be a conjunction
- The first statement has been overused in different communications
- This paragraph is an introduction to a declaration - which should be more direct on the specific claim

### P2:

```
Recent times have seen the Cosmos community grappling with significant challenges stemming from differences about core tokenomics, about the very nature of the $ATOM token (whether it is staking or monetary), about monetization strategy and what types of projects to fund; and there generally appears to be a great cultural chasm that shows no sign of closing about our role and responsibilities as compared to our profit interests. (see [NWV to Prop 848 – $ATOM Must NOT be Money](https://github.com/salmad3/genesis/blob/main/STAKING_VS_MONEY.md)).
```

- Overuse of terms like "times" (refer to prev. paragraph),
- Unclear list presentation, overusing "about"
- Awkward phrasing in the latter part of the sentence

So, I removed the second paragraph due to its perceived redundancy, merging its content with the first and subsequent paragraphs in this section. The first paragraph changed to the following:

```
In the constellations of the Cosmos community, profound challenges have emerged, arising from differing opinions on the core aspects of tokenomics and the intrinsic nature of the $ATOM token—its identity as either a staking or monetary instrument. This period is marked by debates over monetization strategies and the prioritization of project funding.
```

- Tries to incorporate some creativity, specifically "In the constellations of the Cosmos community",
- Aims to provide the "core" direct claim to introduce the declaration, so instead of "such as its vision, mission, tokenomics, architecture..", we replace with "—its identity as either a staking or monetary instrument." The rest can be explored after 
- Intentional phrasing such as "intrinsic nature", "monetization strategies", "instrument"

### P3:

```
Proposal #848 ("halvening") succeeded in getting the required threshold of 50% to pass on Gaia, but a significant portion voted NO or NWV which means that that $ATOM stakers are largely split on the most fundamental tokenomics security design element. 73,165,203ATOM YES vs 56,667,011ATOM NO + 11,669,549ATOM NWV overall YES:NO is 1.07:1. Furthermore, this change was proposed on chain without addressing the valid security concerns raised by the community, with huge errors about the cost of inflation by miscalculating true income, and without a complete halvening schedule, thereby working to undermine hub credibility.
```

- Run-on sentence with multiple clauses lacking clear separation
- General grammatical errors
- Lack of presentable data

### P4:

```
These and prior disagreements have now made clear the need for an alternative hub with a renewed focus and Alignment to serve as the canonical minimal IBC/ICS token hub with respect to Cosmos to champion the ideals of sovereignty, security, and decentralization everywhere; and secondarily to serve as the main base for a political party and more-intelligent voting bloc with respect to Gaia to save Gaia from itself.
```

- Long and convoluted sentence, combining multiple ideas
- Uses of metaphorical language, like "more intelligent about security" and "anti-fragile against even the most powerful of adversaries," which lacks specificity 

Changed to:

```
The passage of Proposal #848, known as "halvening," although achieving 
the required 50% threshold, revealed a deep-seated division among $ATOM 
stakeholders. The voting resulted in a finely balanced YES:NO ratio 
of `1.07:1`:

- **In Favor (YES):** `73,165,203 ATOM`
- **Against (NO):** `56,667,011 ATOM`
- **NoWithVeto (NWV):** `11,669,549 ATOM`

Notable objections of 'NO' voters were the proposal's failure to address 
security concerns before proposing on-chain, its inaccurate inflation 
cost calculations, and the absence of a detailed halvening schedule, 
all of which undermined the Hub's credibility.

Most importantly:

- The voting outcome displayed a significant division within the community 
on the most fundamental security design elemenet.
- A discernible cultural schism has arisen, spotlighting the contrast 
between roles, responsibilities, and profit interests, as detailed 
in [NWV to Prop 848 – $ATOM Must NOT be Money](./STAKING_VS_MONEY.md). 
```

- Aims to provide clean flow
- Generally clarifies the relevant proposal details and key points
- Emphasizes the division within the community more effectively by stating it as a key point, rather than implying it
- Phrases like "Notable objections of 'NO' voters were the proposal's failure to address security concerns before proposing on-chain, its inaccurate inflation cost calculations, and the absence of a detailed halvening schedule" use direct language to specify the reasons for opposition, which were only implied before

```
In light of these unfolding events and persistent disagreements, the 
emergence of an alternative Hub has become necessary. This new entity is 
envisioned to realign and serve as the minimal canonical IBC/ICS token 
Hub within the Cosmos network, upholding the principles of sovereignty, 
security, and decentralization. Further, it aspires to establish a 
political faction and a more discerning voting bloc in relation to Gaia, 
thereby seeking to restore equilibrium and address the internal discord. 

The introduction of a revised distribution model for $ATOM via $ATONE, 
particularly penalizing those who endorsed Proposal #848, is proposed 
to foster a heightened sense of security awareness and fortify resilience 
against even the most formidable adversaries.
```

- Intentional terms like "necessary", "entity"; phrases like "realign", "restore equilibrium" support the claim of "necessity" 
- Provides summary of core principles in three key categories by stating "sovereignty, security, and decentralization"
- In general, a cohesive narrative that leads to the result: "revised distribution model for $ATOM via $ATONE, 
particularly penalizing those who endorsed Proposal #848"

jaekwon commented:
re-opening (but not yet making any determination about whether any splitting is needed; more of a reminder for myself).

giunatale commented on FAQs.md:
```diff
@@ -0,0 +1,21 @@
+architecture. In terms of governance, AtomOne introduces a higher consensus
+threshold (Constitutional Majority) and emphasizes decentralized
+decision-making. In terms of software, AtomOne will start as a fork of
+"cosmoshub4".
```
```suggestion
"cosmoshub-4".
```

giunatale replied:
This is also done in other places. But the correct chain ID has a dash
https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json
